pkg/tfgen/source.go

```diff
@@ -236,6 +249,13 @@ func getDocsPath(repo string, kind DocKind) ([]string, error) {
 
 	var paths []string
 
+	if kind == InstallationDocs {
+		// ${repo}/docs/
```

Comment:
Frontmatter isn't present for repos that use `website`?

Reply:
I'm not sure I understand. Repos that use `website` have front matter: https://raw.githubusercontent.com/hashicorp/terraform-provider-aws/main/website/docs/index.html.markdown

Reply:
This path is only `${repo}/docs/`. I'm saying we probably want to accommodate the `${repo}/website/docs/` path (as is used by your example).